src/main/java/wolox/training/models/Book.java

```diff
@@ -2,11 +2,18 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import jdk.nashorn.internal.objects.annotations.Constructor;
 
 @Entity
 public class Book {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
In Postgres when GenerationType.AUTO is used, it generates an incremental IDs shared between the schema tables. This happens only for this database engine. To generate independent IDs for each table use GenerationType.SEQUENCE.
```suggestion
    @GeneratedValue (strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")            
    @SequenceGenerator(name = "BOOK_SEQ", sequenceName = "BOOK_SEQ")
```